modules/acid/src/main/ts/ephox/acid/gui/components/SaturationBrightnessPalette.ts

```diff
@@ -104,7 +109,8 @@ const paletteFactory = (_translate: (key: string) => string, getClass: (key: str
       dom: {
         tag: 'div',
         attributes: {
-          role: 'presentation'
+          'role': 'presentation',
+          'aria-valuetext': 'Saturation 0%, Brightness 0%'
```

Comment:
Since this has role presentation no screen reader will ever read this. But there really is no role for this looking at what CK is doing they use a slider role here I guess the screen reader will just read the aria-valuetext anyway but it's a bit weird since the user would then never know that they can move in all four directions. Not sure there is a best practices here for this exotic component.

Reply:
Looking at the guidelines they all use three sliders. We do have the manual input fields which serve a similar purpose, so there is still something.

Reply:
This needs to change somehow so either:

1) Change the role to `slider` and pass the hard coded `aira-valuetext` though the translation function and verify that it works on screen readers.
2) Remove the `aria-valuetext` and remove the tab stop for this thing. Tabbing to a presentation element should not be done since the screenreader will likely not pick anything up.